commons/src/main/scala/dagr/commons/CommonsDef.scala

```diff
@@ -62,4 +63,35 @@ object DagrDef {
 
   /** The type of identifier used to uniquely identify tasks tracked by the execution system. */
   type TaskId = BigInt
+
```

Comment:
I really don't think that all these type defs belong in `commons`.  They're very `dagr` specific (`TaskId`), and the paths are even specific to the genomics related tasks.